### ISSUE TITLE
casks/p4merge.rb

### ISSUE THREAD

**Issue by snapfish-soma** [caskroom/homebrew-cask#9147]
sha256 doesn't match

```
openssl md5 -sha256 Downloads/P4V.dmg 
SHA256(Downloads/P4V.dmg)= 766b6f6b8669f889f1186dd96408b5b8af6b9dc6c602784d6d6ea25130007709
```

**Comment by ndr-qef** [comment]
Thanks for the report. The checksum was fixed in 4b4a943a7f.

**Comment by snapfish-soma** [comment]
Thank you @ndr-qef!
